### PR TITLE
Add front-end design for project

### PR DESCRIPTION
**Pull request by etcadinfinitum**
I'm really owning the retro computing aesthetic theme of this event, no regrets :grin:

### CODE REVIEW

File: website/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function renderPage(divName, menuItem) {
+    // hide any visible content divs
+    var contentDivs = document.getElementsByClassName('content');
```
Comment (bpas247) [inline website/main.js]: I would recommend never using `var`. You should use `let` or `const` instead.

File: website/main.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    for (var i = 0; i < contentDivs.length; i++) {
+        if (!contentDivs[i].classList.contains('hidden')) {
+            contentDivs[i].classList.add('hidden');
+        }
+    }
```
Comment (bpas247) [inline website/main.js]: I'm personally a fan of writing declarative JS, so I would use `forEach` here.
```suggestion
    Array.from(contentDivs).forEach(contentDiv => {
        if (!contentDiv.classList.contains('hidden')) {
            contentDiv.classList.add('hidden');
        }
    });
```

File: website/main.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    // make desired div visible
+    var target = document.getElementById(divName);
+    target.classList.remove('hidden');
+
+    // change menu item background to reflect current selection
+    var menuItems = document.getElementsByClassName('menu-item');
+    for (var i = 0; i < menuItems.length; i++) {
+        menuItems[i].classList.remove('active-task');
+    }
```
Comment (bpas247) [inline website/main.js]: Same as above comment.
```suggestion
    Array.from(menuItems).forEach(menuItem => {
        menuItem.classList.remove('active-task');
    });
```

**Review by bpas247 — COMMENTED** [review]
Just a few minor suggestions. Since you already have JQuery as a dependency, I would recommend using that than using vanilla JS 👍

File: website/index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    </div>
+
+    <div id="getCrows" class="content hidden">
+    </div>
+
+    <div id="getFacts" class="content hidden">
+    </div>
+
+    <div id="teachUs" class="content hidden">
+    </div>
```
Comment (bpas247) [inline website/index.html]: Last side note: it's not really best practice to hide DOM elements. If you are wanting to hide them, they should just be removed from the DOM completely. For this use case, we could just use a single DOM element and mutate that whenever the end user click on one of the three menu item elements

Comment (etcadinfinitum) [reply website/index.html]: @bpas247 can you link to code examples or tutorials for this? This is bleeding into the realm of stuff that I don't know how to do, so some examples would be helpful.

Comment (bpas247) [reply website/index.html]: Eh, on second thought this is probably fine. Removing DOM nodes would mess up the layout, and make styling more difficult

**Comment by etcadinfinitum** [comment]
My body is ready for this PR review :toilet: :hankey:

File: website/scripts/populate.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+function showSubmittedFactInfo(response, submission) {
+    // add status message to HTML element
+    let str = "Your request was successful!";
+    if (response['body'] && response['body']['Attributes']) {
+        str += "\nYour input overwrote the input previously added by '" + response['body']['Attributes']['source'] + "', another crow expert.";
+    }
+    let msgDiv = document.getElementById("factRequestPlaceholder");
+    msgDiv.innerHTML = str;
```
Comment (Chris-Johnston) [inline website/scripts/populate.js:68]: should also sanitize here: https://owasp.org/www-community/attacks/xss/

File: website/styles.css [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+#getCrows {
+    background: lavenderblush;
+}
+
+#getFacts {
+    background: lemonchiffon;
+}
+
+#teachUs {
+    background: lightcyan;
```
Comment (Chris-Johnston) [inline website/styles.css]:
```suggestion
    background: azure;
```
`0xE0FFFF` vs `0xF0FFFF`, similar enough

**Review by Chris-Johnston — COMMENTED** [review]
The `pretty.png` file doesn't seem to be used. I'm a bit iffy on adding copyrighted stuff (that isn't ours) to a repo that's MIT licensed.

File: website/404.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Crow Fact Oracle&trade;</title>
```
Comment (Chris-Johnston) [inline website/404.html:9]:
```suggestion
    <title>Crow Fact Oracle</title>
```
Not sure what the `&trade;` is?

Comment (etcadinfinitum) [reply website/404.html:9]: It's the trademark symbol! :D
tm

File: website/styles.css (hunk 6 above)
Comment (etcadinfinitum) [reply website/styles.css]: lol, aight

File: website/scripts/populate.js (hunk 5 above)
Comment (etcadinfinitum) [reply website/scripts/populate.js:68]: There is an open issue for this: #5 

Definitely on the short list of things to do pronto

**Review by Chris-Johnston — APPROVED** [review]

File: website/styles.css (hunk 6 above)
Comment (Chris-Johnston) [reply website/styles.css]: hehe

**Pull request merged**